src/jrd/relations.h

```diff
@@ -734,3 +734,10 @@ RELATION(nam_config, rel_config, ODS_13_0, rel_virtual)
 	FIELD(f_cfg_is_set, nam_cfg_is_set, fld_cfg_is_set, 0, ODS_13_0)
 	FIELD(f_cfg_source, nam_cfg_source, fld_file_name2, 0, ODS_13_0)
 END_RELATION
+
+//// FIXME: ODS version
+// Relation 54 (RDB$KEYWORDS)
+RELATION(nam_keywords, rel_keywords, ODS_13_0, rel_virtual)
+	FIELD(f_keyword_name, nam_keyword_name, fld_keyword_name, 0, ODS_13_0)
+	FIELD(f_keyword_reserved, nam_keyword_reserved, fld_keyword_reserved, 0, ODS_13_0)
```

Comment:
I'm not sure about the column names: `RDB$KEYWORD_NAME`, `RDB$KEYWORD_RESERVED`.

As usual, we are repetitive and maybe not well spelled in case of "KEYWORD RESERVED". I did like `RDB$CONFIG` columns where every column is prefixed with `RDB$CONFIG_`

Reply:
I don't have any strong preference about that, as we always had RDB$RELATION_NAME inside RDB$RELATIONS and so on ;-) With just two fields in RDB$KEYWORDS I don't see it as a problem.

Reply:
It's not clear if you mean you prefer the prefix maintained in both columns or something different.

Reply:
Sorry for confusion, I'm OK with prefix in both columns.